### PR TITLE
Test for STAR GenomeGenerate

### PR DESCRIPTION
**Pull request by J35P312**
Howdie!
I have made a test file for star genome generate (i.e to generate the index). 
//Jesper

### CODE REVIEW

**Comment by J35P312** [comment]
Aloha!
I have completed the changes to the code:

I added the star generate test to the list of tests, removed un-needed the star_genome_generate if cases, and fixed the outputs of star_genome_generate tests!

File: t/star_genome_generate.t [hunk 1]
```diff
@@ -120,20 +120,41 @@ my %base_argument = (
 
 my %required_argument = (
     fasta_path => {
-        input           => q{test_file.fasta},
-        expected_output => q{test_file.fasta},
+        input           => catfile(qw{ dir test_file.fasta}),
```
Comment (henrikstranneheim) [inline t/star_genome_generate.t]: fasta}) -> fasta })

Comment (henrikstranneheim) [reply t/star_genome_generate.t]: Same syntax also need corrections downstream

File: t/star_genome_generate.t [hunk 2]
```diff
@@ -120,20 +120,41 @@ my %base_argument = (
...
+        expected_output => q{--genomeFastaFiles} . $SPACE . catfile(qw{ dir test_file.fasta}),
     },
     genome_dir_path => {
-        input           => q{genome_dir_path},
-        expected_output => q{genome_dir_path},
+        input           => catfile(qw{ dir genome_dir_path}),
+        expected_output => q{--genomeDir} . $SPACE . catfile(qw{ dir genome_dir_path}),
```
Comment (henrikstranneheim) [inline t/star_genome_generate.t]: Same here _path})

File: t/star_genome_generate.t [hunk 3]
```diff
@@ -120,20 +120,41 @@ my %base_argument = (
...
     },
     gtf_path => {
-        input           => q{test_gtf.gtf},
-        expected_output => q{test_gtf.gtf},
+        input           => catfile(qw{ dir test_gtf.gtf}),
```
Comment (henrikstranneheim) [inline t/star_genome_generate.t]: And here

File: t/star_genome_generate.t [hunk 4]
```diff
@@ -120,20 +120,41 @@ my %base_argument = (
...
+        expected_output => q{--sjdbGTFfile} . $SPACE . catfile(qw{ dir test_gtf.gtf}),
     },
 );
 
-my %specific_argument = ();
+my %specific_argument = (
+    fasta_path => {
+        input           => catfile(qw{ dir test_file.fasta}),
+        expected_output => q{--genomeFastaFiles} . $SPACE . catfile(qw{ dir test_file.fasta}),
```
Comment (henrikstranneheim) [inline t/star_genome_generate.t]: And same syntax error repeated in specific_argument

**Comment by J35P312** [comment]
Woopsie daisy! Now I have fixed the syntax error! =P

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**